randovania/games/samus_returns/gui/preset_settings/msr_patches_tab.py

```diff
@@ -1,37 +1,37 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+import typing
 
 from PySide6 import QtWidgets
 
 from randovania.games.samus_returns.layout import MSRConfiguration
+from randovania.gui.generated.preset_msr_patches_ui import Ui_PresetMSRPatches
```

Comment:
Is mypy running before creating the ui files or why it is complaining?

Reply:
it is, I saw it recently. Gotta fix it later